core/math/a_star_grid_2d.cpp

```diff
@@ -234,14 +232,14 @@ AStarGrid2D::Point *AStarGrid2D::_jump(Point *p_from, Point *p_to) {
 		return p_to;
 	}
 
-	int64_t from_x = p_from->id.x;
-	int64_t from_y = p_from->id.y;
+	int from_x = p_from->id.x;
+	int from_y = p_from->id.y;
```

Comment:
The change make sense, but I wonder if it should be changed to `int32_t` rather than simple int for consistency with a Vector.

Reply:
I'm not sure. In theory the size of `int` can be different, but in practice we use these types interchangeably in the codebase. I replaced `int` with `int32_t` where it implies point coordinates. Let's see if this will pass CI (`is_in_bounds()` is exposed for GDExtension).